### DISCUSSION TITLE
Generate CRUD-app from Firebase

### DISCUSSION THREAD

**Discussion by drsounds** [Budibase/budibase#8540] [Help] [answered]
I am considering Budibase low code platform for our company, and today we currently use Firebase as our database solution. I was glad to see this wonderful project and support for Firebase, but it seems I can't use Firebase as a regular 'Table' for creating the CRUD-functionality with ease.
I tried to add all four types of queries (Create, Read, Update, Delete) but Firebase tables can't still be used as a CRUD-data source.
Is it something I've missed or is this a feature work in progress?

**Comment by melohagan** [comment, marked as answer]
Hey @drsounds 

The Firebase connector allows you to perform [custom-queries](https://docs.budibase.com/docs/custom-queries) however it does not have full **Data** tab table integration. 
This is work to be done, but it's not likely to be available in the near future.
